### ISSUE TITLE
bfloat16 numpy not supported

### ISSUE THREAD

**Issue by sh1man** [kwon-evan/LPRNet#5]
You had it written utils.py
`def accuracy(logits, labels, lengths, chars):
    preds = logits.cpu().detach().numpy()`

I changed it to `logits.to(torch.float32).cpu().detach().numpy()`
this is normal practice ?
